src/utils/crypto.util.ts

```diff
@@ -1,16 +1,45 @@
+import scrypt from 'react-native-scrypt';
 import {
   secureCellSealWithPassphraseDecrypt64,
   secureCellSealWithPassphraseEncrypt64,
   symmetricKey64
 } from 'react-native-themis';
 import { from, Observable, of } from 'rxjs';
-import { catchError } from 'rxjs/operators';
+import { map, switchMap, catchError } from 'rxjs/operators';
 
 export interface EncryptedData {
   encrypted64: string;
   salt64: string;
 }
 
+const scrypt$ = async (password: string, salt: string) =>
+  from<string>(
+    await scrypt(
+      Buffer.from(password, 'base64').toString(),
+      Buffer.from(salt, 'base64').toString(),
+      65536,
+      8,
+      1,
+      32,
+      'base64'
```

Comment:
👀

Reply:
@vixentael  is everything ok in scrypt-call signature?